src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,7 @@
+spring.jpa.database=POSTGRESQL
+spring.datasource.platform=postgres
+spring.datasource.url=${DATABASE_URL}
```

Comment:
DATABASE_URL is an env far setup by heroku to point to our db in prod. This url changes from time to time but our data is kept intact. No need to change it, but know what it is and where it's coming from.